### PR TITLE
Fixes to custom service doc

### PR DESCRIPTION
**Pull request by vrakesh**
Before or while filing an issue please feel free to join our [ slack channel](https://join.slack.com/t/mms-awslabs/shared_invite/enQtNDk4MTgzNDc5NzE4LTBkYTAwMjBjMTVmZTdkODRmYTZkNjdjZGYxZDI0ODhiZDdlM2Y0ZGJiZTczMGY3Njc4MmM3OTQ0OWI2ZDMyNGQ) to get in touch with development team, ask questions, find out what's cooking and more!

## Testing done:

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/custom_service.md [hunk 1]
```diff
@@ -100,7 +112,7 @@ def handle(data, context):
     if data is None:
         return None
 
-    return _service.handle(data, context)
+    return _service.handler(data, context)
```
Comment (vdantu) [inline docs/custom_service.md]: Call this handle. The text below references to handle().

File: docs/custom_service.md [hunk 2]
```diff
@@ -72,15 +72,16 @@ class ModelHandler(object):
         :return: list of preprocessed model input data
         """
         assert self._batch_size == len(batch), "Invalid input batch size: {}".format(len(batch))
-        return None
+        return batch
```
Comment (vdantu) [inline docs/custom_service.md]: This seems like a dummy template. Why changes this from None to something else?

File: docs/custom_service.md [hunk 3]
```diff
@@ -72,15 +72,16 @@ class ModelHandler(object):
...
 
     def inference(self, model_input):
         """
         Internal inference methods
         :param model_input: transformed model input data
         :return: list of inference output in NDArray
         """
-        return None
+        # Do some inference call to engine here and return output
+        return model_input
```
Comment (vdantu) [inline docs/custom_service.md]: Same here. Whats the point here?

File: docs/custom_service.md [hunk 4]
```diff
@@ -89,6 +90,17 @@ class ModelHandler(object):
         :return: list of predict results
         """
         return ["OK"] * self._batch_size
+        
+    def handler(self, data, context):
```
Comment (vdantu) [inline docs/custom_service.md]: Call this handle()

Comment (vrakesh) [reply docs/custom_service.md]: Yeah makes sense, I thought of differentiating it a bit, and I agree it can be a dummy

**Comment by vrakesh** [comment]
Done added comments to the preprocess and postprocess as well

**Pull request merged**